src/ert/storage/local_storage.py

```diff
@@ -74,10 +74,13 @@ def __init__(
         self._index: _Index
 
         if self.can_write:
-            self.path.mkdir(parents=True, exist_ok=True)
-            self._migrate(ignore_migration_check)
-            self._index = self._load_index()
+            if not any(self.path.glob("*")):
+                # No point migrating if storage does not exist
```

Comment:
`if not self.path.exists()` ?

If you are checking whether the directory does not exist _or is empty_ then I think `.glob("*")` is fine (alternatively, `os.listdir(path)`) but then the comment is wrong.

Reply:
Yes, was checking if empty, will update comment 👍